src/py/flwr/server/app.py

```diff
@@ -388,6 +417,59 @@ def run_superlink() -> None:
         driver_server.wait_for_termination(timeout=1)
 
 
+def _try_setup_client_authentication(
+    args: argparse.Namespace,
+    certificates: Optional[Tuple[bytes, bytes, bytes]],
+) -> Optional[Tuple[Set[bytes], ec.EllipticCurvePublicKey, ec.EllipticCurvePrivateKey]]:
+    if args.require_client_authentication:
+        if certificates is None:
+            sys.exit(
+                "Certificates are required to enable client authentication. "
```

Comment:
I assume this is a duplicated text as the below lines are telling essentially the same.
I suggest using this message for both certificate, privatkey and/or certificate authority.

```
"Please provide a valid certificate and private key "
-> {print the --help content here} 
```
This could then also be used in [L433-L436](https://github.com/adap/flower/pull/2948/files#diff-30c865cb167adb96d6ba27ee5a7bdd4b0cb5a95ab52bdda95080da549a58dc4aR433-R436)

Reply:
It's not a duplicate, certificates are for a secure channel (SSL/TLS) and elliptic curve key pair is needed for authentication.